packages/bundles/src/all-imports.js

```diff
@@ -0,0 +1,90 @@
+/* Core */
+import '@vaadin/accordion';
+import '@vaadin/app-layout';
+import '@vaadin/app-layout/vaadin-drawer-toggle.js';
+import '@vaadin/avatar';
+import '@vaadin/avatar-group';
+import '@vaadin/button';
+import '@vaadin/checkbox';
+import '@vaadin/checkbox-group';
+import '@vaadin/combo-box';
+import '@vaadin/context-menu';
+import '@vaadin/custom-field';
+import '@vaadin/date-picker';
+import '@vaadin/date-time-picker';
+import '@vaadin/details';
+import '@vaadin/dialog';
+import '@vaadin/email-field';
+import '@vaadin/form-layout';
+import '@vaadin/form-layout/vaadin-form-item.js';
+import '@vaadin/grid';
```

Comment:
You could use `@vaadin/grid/all-imports.js` to remove individual helper components imports.